server/store/datastore/org.go

```diff
@@ -28,13 +28,24 @@ func (s storage) OrgCreate(org *model.Org) error {
 }
 
 func (s storage) orgCreate(org *model.Org, sess *xorm.Session) error {
-	// sanitize
-	org.Name = strings.ToLower(org.Name)
+	// Get all organizations to check for conflicts (this should actually be impossible in the forge but better double-check)
+	orgs, err := s.OrgList(nil)
+	if err != nil {
+		return err
+	}
+
+	for _, existingOrg := range orgs {
+		if strings.EqualFold(existingOrg.Name, org.Name) {
+			return fmt.Errorf("organization name already exists")
+		}
+	}
```

Comment:
We store orgs now case-sensitive in the DB, why do we check here for case-insensitive duplicates?